docs/source/tutorial_mturk.md

```diff
@@ -10,15 +10,9 @@ instructions for hosting a ParlAI-Chat style task can be found
 and regular collection tasks can be found
 [here](https://github.com/facebookresearch/Mephisto/tree/master/examples/static_react_task).
 
-We will be moving some of our common tasks out of the Mephisto repo into
```

Comment:
change the box to be yellow or red now, plz

Reply:
Makes sense, changing to yellow warning